module/Application/src/View/Helper/Breadcrumbs.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Application\View\Helper;
+
+use Laminas\View\Helper\Placeholder\Container\{
+    AbstractContainer,
+    AbstractStandalone,
+};
+
+/**
+ * Helper for setting and retrieving breadcrumbs.
+ */
+class Breadcrumbs extends AbstractStandalone
+{
+    public function __construct()
+    {
+        return $this;
+    }
+
+    public function addBreadcrumb(
+        string $breadcrumb = '',
+        bool $active = true,
+        string $url = '',
+        ?string $setType = null,
+    ) {
+        if (null === $setType) {
+            $setType = AbstractContainer::APPEND;
+        }
+
+        if ('' !== $breadcrumb) {
+            $item = [
+                'name' => $breadcrumb,
+                'active' => $active,
+                'url' => $url,
+            ];
+
+            if (AbstractContainer::SET === $setType) {
+                $this->set($item);
```

Comment:
Dit is een beetje hacky. Ik zie ook niet waar de methods set, prepend en append gedefineerd worden.

Reply:
Dat komt omdat Laminas het raar heeft gedaan, we extenden dus `AbstractStandalone` en die gebruikt (stiekem) weer `AbstractContainer` waar de functies wel gedefinieerd zijn.

Reply:
Alle Laminas view helpers die dit gebruiken doen het op die manier (exacte implementatie verschilt natuurlijk):

https://github.com/laminas/laminas-view/blob/01cfed29007a26873a8b9d35f93bd7fbc0512804/src/Helper/HeadLink.php#L90-L105